src/Symfony/Component/Console/Question/ChoiceQuestion.php

```diff
@@ -22,7 +22,7 @@ class ChoiceQuestion extends Question
 {
     private $choices;
     private $multiselect = false;
-    private $prompt = ' > ';
+    private $prompt = null;
```

Comment:
this is a BC break, as `getPrompt` can now return `null`.

Reply:
Fixed